pages/contribute.tsx

```diff
@@ -19,12 +31,123 @@ function Contribute(): JSX.Element {
             site_name: 'open source at ACM at UCLA',
           }}
         />
-        <h1>
-          get started
-        </h1>
+        <div className='row spaced-row'>
+          <h1>get started </h1>
+          <a href='#good-first-issues'><button>good first issues</button></a>
+        </div>
+        <hr />
+        <h2>
+          what is open source?
+        </h2>
+        <p>
+          Open source software is code that is made freely available for <b>anyone</b> to
+          use or enhance! Here at ACM, we take pride in <b>all</b> of our projects being open-source
+          and open for anyone to work on.
+        </p>
+        <h2>
+          what makes open source possible?
+        </h2>
+        <p>
+          The main back bone of open source initiatives are tools that can make collaboration and sharing of code as
+          simple as possible. Git and <ELink link='https://github.com/'>Github</ELink> are two really popular tools
+          that do just that used by companies like Google, Meta, and more, and is what we use at ACM as well.
```

Comment:
Copy suggestion; mostly on conciseness.

```suggestion
          The main backbone of open source initiatives are tools that simplify collaboration and sharing of code. Git and <ELink link='https://github.com/'>Github</ELink> are two popular tools used by researchers, companies, and students; at ACM, we use both.
```